src/events/index.js

```diff
@@ -0,0 +1,29 @@
+const _ = require('lodash');
+
+const eventType = require('./types');
+
+function resolveHandler(event) {
+  // eslint-disable-next-line
+  const handler = require(`./handlers/${event}`);
```

Comment:
probably we should add a `try..catch` around to post human friendly error if file is not found